extensions/pipe-commands/src/types.ts

```diff
@@ -35,5 +35,5 @@ export type ScriptArgument = {
   percentEncoded?: boolean;
 };
 
-export type InputType = "text" | "file" | "url";
+export type InputType = "text" | "file" | "url" | "clipboard";
```

Comment:
I don't think the clipboard input is necessary. What does it add on top of text ?

Reply:
Could you check this @iloveitaly